src/validation_workflow/rpm/validation_rpm.py

```diff
@@ -75,6 +72,7 @@ def cleanup(self) -> bool:
         try:
             for project in self.args.projects:
                 execute(f'sudo systemctl stop {project}', ".")
+                execute(f'sudo yum remove {project} -y', ".")
         except:
-            raise Exception('Failed to Stop Cluster')
+            raise Exception('Exception occurred either while attempting to stop cluster or while removing opensearch')
```

Comment:
This can be OS or OSD so keep the message generic `{project}` would be the right parameter here

Reply:
Sure Sayali, will change it